packages/dx-react-grid/docs/reference/table-row-detail.md

```diff
@@ -13,57 +13,57 @@ A plugin that renders a table detail row.
 
 Name | Type | Default | Description
 -----|------|---------|------------
-template | (args: [DetailContentArgs](#detail-content-args)) => ReactElement | | A component that renders row details.
-detailCellTemplate | (args: [DetailCellArgs](#detail-cell-args)) => ReactElement | | A component that renders a detail cell.
-detailToggleCellTemplate | (args: [DetailToggleArgs](#detail-toggle-args)) => ReactElement | | A component that renders the detail toggle control.
-detailRowTemplate | (args: [DetailRowArgs](#detail-row-args)) => ReactElement | | A component that renders a detail row.
-detailToggleCellWidth | number | | Specifies the detail toggle cell width.
-rowHeight | number &#124; string | 'auto' | Specifies the detail row height.
+contentComponent | ElementType&lt;[DetailContentProps](#detailcontentprops)&gt; | | A component that renders row details.
+cellComponent | ElementType&lt;[TableDetailCellProps](#tabledetailcellprops)&gt; | | A component that renders a detail cell.
+rowComponent | ElementType&lt;[TableDetailRowProps](#tabledetailrowprops)&gt; | | A component that renders a detail row.
+toggleCellComponent | ElementType&lt;[TableDetailToggleCellProps](#tabledetailtogglecellprops)&gt; | | A component that renders the detail toggle control.
```

Comment:
toggle**Cell**Component - A component that renders the detail **toggle control**. 
It isn't clear what this component renders cell or toggle control.